pylibs/otns/cli/OTNS.py

```diff
@@ -455,6 +464,7 @@ def get_panid(self, nodeid: int) -> int:
         Get node pan ID.
 
         :param nodeid: node ID
+
         :return: pan ID
         """
```

Comment:
Not implemented yet? I think we have a syntax error here, we should add `pass` at least. Better if we can raise exception or add an assertion.

Reply:
Good catch!
But it's not syntax error since we have doc string which can also be seen as a statement.